app/views/categories/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="category">
+  <h2>Name: <%= @category.name %></h2>
+  <% if @category.items %>
+    <h3>Items:</h3>
+    <% @category.items.each do |item| %>
```

Comment:
I'll start to add some helpers for things like this, also id like to make some of these partials after we build the functionality